src/gui/modalMenu.cpp

```diff
@@ -306,13 +306,13 @@ bool GUIModalMenu::preprocessEvent(const SEvent &event)
 
 			s32 selected_idx = dropdown->getSelected();
 			s32 option_size = dropdown->getItemCount();
-			std::string list_of_options[option_size];
+			std::vector<std::string> list_of_options;
 
 			for (s32 i = 0; i < option_size; i++) {
-				list_of_options[i] = wide_to_utf8(dropdown->getItem(i));
+				list_of_options.push_back(wide_to_utf8(dropdown->getItem(i)));
```

Comment:
it's mostly the same but does more allocations, why change this ?

Reply:
like the commit says: to avoid VLAs

Reply:
Getting rid of these allocations is 100% premature optimization.